storage/connect/reldef.cpp

```diff
@@ -23,6 +23,9 @@
 #include <dlfcn.h>          // dlopen(), dlclose(), dlsym() ...
 #include "osutil.h"
 //#include "sqlext.h"
+#if defined(__OpenBSD__)
+#define RTLD_NOLOAD 0
+#endif
 #endif
```

Comment:
ok, got  it. although, I'd rather remove RTLD_NOLOAD from the code below, I don't see any need for it.

Reply:
It is needed because OpenBSD does not have this flag.